README.rst

```diff
@@ -1,2 +1,92 @@
-layered-config
+layered_config
 ==============
+
+A simple library to ease maintaining groups of similar configurations:
+   * Keep config files DRY like code.
+   * Allow environment variables to override configuration file values.
+   * `munch`_ your loaded configuration for easy field access.
+
+Instead of copy'n'paste-ing related configuration files for different situations,
+refactor the configuration sections and keys into separate smaller configuration files.
+Then use this library to combine those files in various ways to produce a single |ConfigParser| object,
+as if it had come from a single file.
+
+
+tl;dr Quick Start in 3 Easy Steps:
+----------------------------------
+
+1. Create some configs:
+~~~~~~~~~~~~~~~~~~~~~~~
+
+For this simple example, consider three config files:
+
+    | ``base.config`` - all the common stuff between your environments.
+    | ``staging.config`` - config info specific to your staging environment.
+    | ``demo.config`` - config info specific to your demo environment.
+
+Create a ``master.config`` file describing how to layer the simpler config files.
+The master config file's name is arbitrary.
+Your code needs to know its name, but this library doesn't care.
+
+``master.config`` file:
+
+.. code:: INI
+
+    [env1]
+    # layers is a comma separated list of configuration files to load,
+    # in order, from left to right.
```

Comment:
This tells me import order but doesn't tell me priority (do newer files overwrite older or vice-versa?)

Reply:
Oops, yeah, will fix.